medicines/doc-index-updater/src/service_bus_client/mod.rs

```diff
@@ -102,6 +102,10 @@ pub enum ProcessMessageError {
     SftpError(#[from] SftpError),
     #[error("Cannot find document with ID {0}")]
     DocumentNotFoundInIndex(String),
+    #[error("Cannot delete blob with ID {0}: {1}")]
+    FailedDeletingBlob(String, String),
+    #[error("Cannot restore index for blob {0}: {1}")]
```

Comment:
Seems odd that one error message says `"blob with ID <id>"` and the other says `"blob <id>"`. I'd go for the latter in both.

Reply:
True - added.